### PR TITLE
Fix cm/secret templates with checks for CR values instead of chart values

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

File: helm/appcatalog-chart/values.yaml [hunk 1]
```diff
@@ -21,6 +21,5 @@ configMap:
   values:
 
 ### Example Secret configuration structure ###
-#secret:
-#  values:
-#    token: ""
+secret:
+  values:
```
Comment (rossf7) [inline helm/appcatalog-chart/values.yaml]: Nesting is still wrong here. Can you move it under `.Values.appCatalog.config` so it's less confusing?

Comment (corest) [reply helm/appcatalog-chart/values.yaml]: Ok. What is with `.Values.configMap.values` then? Is that a valid thing?

Comment (corest) [reply helm/appcatalog-chart/values.yaml]: e.g. https://github.com/giantswarm/appcatalog/blob/4866edf8a588956032b39f7e0e4d93a0bd72bcd1/helm/appcatalog-chart/values.yaml

Comment (rossf7) [reply helm/appcatalog-chart/values.yaml]: Yes `.Values.configMap.values` isn't used either. I think it makes sense to move both under `.Values.appCatalog.config`.

**Pull request merged**